src/components/Vagas Abertas/InfoVagas.css

```
@@ -0,0 +1,29 @@
+.about-details {
+  display: flex;
+  flex-direction: column;
+  width: 768px;
+  padding: 24px 24px 28px 24px;
```

o `24px` setado no meio, vai pra horizontal inteira

```suggestion
  padding: 24px 24px 28px;
```